README.md

```diff
@@ -43,41 +38,70 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
 
 ### Addons outside of Scratch Addons itself
 
-Other extensions (and even forks of Scratch) can also provide their users with most addons from Scratch Addons, as long as a compatibility layer for `addon.*` and other parts of the addon loader is present. Notable examples are the [Scratch 3 Developer Tools extension](https://github.com/ScratchAddons/DevtoolsExtension) and the [TurboWarp](https://github.com/TurboWarp/scratch-gui/tree/develop/src/addons) editor. These also inherit translations from the Scratch Addons project.
+Other extensions and even forks of Scratch can also provide their users with most addons from Scratch Addons, as long as a compatibility layer for `addon.*` and other parts of the addon loader are present. A notable example is the [TurboWarp](https://github.com/TurboWarp/scratch-gui/tree/develop/src/addons) editor. These also inherit translations from the Scratch Addons project.
+
+### Structure
+
+- `.github`: GitHub templates, workflows and contributing files.
+- `_locales`: Translations for the settings page, popup and everything else except addons.
+- `addon-api`: The code for the `addon-*` APIs.
+- `addons-l10n`: Translations for strings userscripts add to a page. Each addon has its own file.
+- `addons`: The addons themselves. Each addon has it own directory with an `addon.json` and an entry in `addons.json`.
+- `background`: Background scripts for handling things such as addon settings.
+- `content-scripts`: The main extension script and addon injector.
+- `images`: Logos, screenshots and icons. Images only used by addons go in that addon's directory instead.
+- `libraries`: Libraries for things such as handling the global theme. There are also some third-party libraries.
```

Comment:
I'm not that familiar with these parts of the extension. Could someone else come up with better explanations?